homeassistant/components/todoist/calendar.py

```diff
@@ -117,18 +115,13 @@ async def async_setup_platform(
 
     # Look up IDs based on (lowercase) names.
     project_id_lookup = {}
-    label_id_lookup = {}
```

Comment:
This lookup dict isn't needed anymore since we pass the labels as strings instead of an array of ids in the service call.